.github/workflows/codeql.yml

```
@@ -40,10 +40,10 @@ jobs:
       python: ${{ steps.filter.outputs.python }}
     steps:
       - name: Checkout repository
-        uses: it-at-m/lhm_actions/action-templates/actions/action-checkout@691ffa2542ed7cc1c53c416d22ace9a153d5152f # v1.0.21
+        uses: it-at-m/lhm_actions/action-templates/actions/action-checkout@12966547963631f8429d42bab860485b413003c2 # v1.0.22
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)